products/mssqlserver.md

```diff
@@ -8,6 +8,12 @@ activeSupportColumn: true
 versionCommand: select @@version
 releaseDateColumn: true
 releases:
+-   releaseCycle: "2022"
+    releaseLabel: "2022 RTM"
```

Comment:
LGTM except for the "RTM" label. On https://endoflife.date/mssqlserver only CU (or SP for older versions) are tracked in the release label. No cumulative update has been done so far for SQL Server 2022, so I think the release label should be `2022` for the time being. Do you agree with my reasoning @BiNZGi ?

Reply:
Yes, I agree, I'll change that :+1: